### PR TITLE
TV UX improvements

### PR DESCRIPTION
**Pull request by self-similarity**
- Added auto focus to the skip timestamp popup on TV

- Added a checkmark to the Dub/Sub/Season/Episode selector

### CODE REVIEW

File: app/src/main/java/com/lagradost/cloudstream3/ui/result/SelectAdaptor.kt [hunk 1]
```diff
@@ -45,19 +45,9 @@ class SelectAdaptor(val callback: (Any) -> Unit) : RecyclerView.Adapter<Recycler
         if(newIndex == selectedIndex) return
         val oldIndex = selectedIndex
         selectedIndex = newIndex
-        recyclerView.apply {
```
Comment (self-similarity) [inline app/src/main/java/com/lagradost/cloudstream3/ui/result/SelectAdaptor.kt:48]: This failed on edgecases in the recyclerview.
Although not as efficient it is easier to only update the items for guaranteed results.

File: app/src/main/res/color/button_selector_color.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_selected="true" android:state_focused="true" android:color="@color/black"/>
```
Comment (LagradOst) [inline app/src/main/res/color/button_selector_color.xml:3]: Have you tested this on white mode?

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -737,6 +737,11 @@
         <item name="android:layout_marginEnd">0dp</item>
     </style>
 
+    <style name="SelectableButtonTV" parent="RegularButtonTV">
+        <item name="icon">@drawable/cpv_preset_checked</item>
```
Comment (LagradOst) [inline app/src/main/res/values/styles.xml:741]: Is this the correct checkmark drawable?

**Review by LagradOst — APPROVED** [review]
LGTM, just wondering some stuff

File: app/src/main/res/color/button_selector_color.xml (hunk 2 above)
Comment (self-similarity) [reply app/src/main/res/color/button_selector_color.xml:3]: I have not, I will do

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (self-similarity) [reply app/src/main/res/values/styles.xml:741]: Will double check

File: app/src/main/res/color/button_selector_color.xml (hunk 2 above)
Comment (self-similarity) [reply app/src/main/res/color/button_selector_color.xml:3]: It works, although white mode is generally unusable on TV.

**Comment by self-similarity** [comment]
Ready for merge :+1:

**Pull request merged**